src/libraries/tests.proj

```diff
@@ -7,6 +7,9 @@
     <CoverageReportDir>$(ArtifactsDir)coverage</CoverageReportDir>
     <SerializeProjects>true</SerializeProjects>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp5.0</TargetFramework> 
```

Comment:
Is this required for P2P to work?

Reply:
SDK requires TargetFramework to set TargetFrameworkIdentifier as this property cant be empty.
There is no use for it other than that